src/anomalib/models/efficient_ad/torch_model.py

```diff
@@ -148,8 +149,8 @@ def __init__(self, out_channels, padding, img_size, *args, **kwargs) -> None:
         super().__init__(*args, **kwargs)
         self.img_size = img_size
         self.last_upsample = (
-            int(img_size[0] / 4) if padding else int(img_size[0] / 4) - 8,
-            int(img_size[1] / 4) if padding else int(img_size[1] / 4) - 8,
+            math.ceil(img_size[0] / 4) if padding else math.ceil(img_size[0] / 4) - 8,
+            math.ceil(img_size[1] / 4) if padding else math.ceil(img_size[1] / 4) - 8,
```

Comment:
Could you please add a comment here about why this is needed exactly. I think that would help down the line if anything else needs to be changed.

Reply:
added the comment, that ceil is necessary to match PDN output shape

Reply:
Okay, thanks.